.eslintrc.json

```diff
@@ -96,6 +96,13 @@
         "fp/no-delete": 0
       }
     },
+    {
+      "files": ["**/*.md"],
+      "rules": {
+        "no-undef": 0,
+        "no-unused-vars": 0
```

Comment:
Markdown examples sometimes only show a variable declaration, but not where it's being used.